lib/pages/splash/splash_page.dart

```diff
@@ -1,12 +1,151 @@
+import 'dart:async';
+import 'dart:io';
+import 'dart:convert';
+
 import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:provider/provider.dart';
+import 'package:path_provider/path_provider.dart';
+import 'package:campus_app/core/settings.dart';
+import 'package:campus_app/core/themes.dart';
+import 'package:campus_app/pages/home/home_page.dart';
+
+class SplashPage extends StatefulWidget {
+  static const routeName = '/splash';
 
-class SplashPage extends StatelessWidget {
   const SplashPage({Key? key}) : super(key: key);
 
+  @override
+  State<SplashPage> createState() => _SplashPageState();
+}
+
+class _SplashPageState extends State<SplashPage> {
+  Stopwatch loadingTimer = Stopwatch();
+  Stopwatch idleTimer = Stopwatch();
+  String _directoryPath = '';
+
+  Settings? loadedSettings;
+
+  void startApp(BuildContext context) {
+    if (loadedSettings != null) {
+      // Normal app start
+      print('Initiate normal app start.');
```

Comment:
We should avoid print-Statements in production code. For debugging we could use `debugPrint()` instead, which provide some more (useful) information. Lets check out: [https://medium.com/@abhishekbedi/print-vs-debugprint-eac33a774d5a](https://medium.com/@abhishekbedi/print-vs-debugprint-eac33a774d5a)

Reply:
Every `print()` should now be replaced with `debugPrint()`.